src/shared/sidebar/sidebar.css

```diff
@@ -0,0 +1,56 @@
+.sidebar-container {
+    position: absolute;
+    z-index: 2;
+    padding: 0rem 2rem 2rem 1rem;
+    margin: -1rem 0 0 -1rem;
+    border-bottom-right-radius: 2rem;
+    border-right: 2px solid var(--blueviolet);
+    border-bottom: 2px solid var(--blueviolet);
+}
+
+.link-color {
+    margin-top: 2rem;
+}
+
+.link-color:hover {
+    color: var(--primary);
+}
+
+.sidebar-icon {
+    font-size: 1.4rem;
+}
+
+.sidebar-text {
+    margin-left: 1rem;
+    font-size: 1.1rem;
+    font-weight: 600;
```

Comment:
use the class you have created in index.css

Reply:
Yes yes, I missed